fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -89,8 +91,9 @@ struct GrvProxyStats {
 	    txnBatchPriorityStartOut("TxnBatchPriorityStartOut", cc),
 	    txnDefaultPriorityStartIn("TxnDefaultPriorityStartIn", cc),
 	    txnDefaultPriorityStartOut("TxnDefaultPriorityStartOut", cc), txnThrottled("TxnThrottled", cc),
-	    transactionRateAllowed(0), batchTransactionRateAllowed(0), transactionLimit(0), batchTransactionLimit(0),
-	    percentageOfDefaultGRVQueueProcessed(0), percentageOfBatchGRVQueueProcessed(0),
+	    getRateInfo("GetRateInfo", cc), leaseTimeout("LeaseTimeout", cc), systemGRVQueueSize(0), defaultGRVQueueSize(0),
```

Comment:
Nit: I think we should pluralize lease timeouts (this requires changes elsewhere too):

```suggestion
	    getRateInfo("GetRateInfo", cc), leaseTimeouts("LeaseTimeouts", cc), systemGRVQueueSize(0), defaultGRVQueueSize(0),
```